mmdet/ops/corner_pool/corner_pool.py

```diff
@@ -92,10 +93,29 @@ class CornerPool(nn.Module):
         'top': TopPoolFunction,
     }
 
+    cummax_dim_flip = {
+        'bottom': [2, False],
+        'left': [3, True],
+        'right': [3, False],
+        'top': [2, True],
+    }
+
     def __init__(self, mode):
         super(CornerPool, self).__init__()
         assert mode in self.pool_functions
-        self.corner_pool = self.pool_functions[mode]
+        if torch.__version__ >= '1.5.0':
+            self.corner_pool = self.cummax_dim_flip[mode]
```

Comment:
This is quite weird. I suggest saving `mode` and get dim and flip in `forward()`.